src/server/getters/get-market-info.ts

```diff
@@ -52,8 +53,16 @@ export function reshapeMarketsRowToUIMarketInfo(row: MarketsRow, outcomesInfo: A
   return marketInfo;
 }
 
+export function getMarketsWithReportingState(db: Knex, selectColumns?: Array<string>): Knex.QueryBuilder {
```

Comment:
@pgebheim thoughts about the right place to put this? Feels funny to put it in another getter's file, but other meta-query-related things were already here and included from other getters.

Reply:
I agree having these meta-getters and reorganization functions inside the getter file (and/or equivalent things inside log-processor files) feels "off"... Maybe we should put them in a separate "helpers" directory?  Another (maybe more consistent with our overall organization) option is, we could change get-market-info.ts into a directory with the main getMarketInfo function as index.ts, and one helper function per file.

For now I think it's fine to put it here in the get-market-info.ts file though!